packages/excalidraw/components/App.tsx

```diff
@@ -7419,7 +7417,18 @@ class App extends React.Component<AppProps, AppState> {
               event[KEYS.CTRL_OR_CMD] ? null : this.state.gridSize,
             );
 
-          this.maybeSuggestBindingForAll(selectedElements);
+          selectedElements.filter(isLinearElement).forEach((element) => {
+            const candidateBindables =
+              getOriginalBindingsIfStillCloseOfLinearElement(
+                element,
+                elementsMap,
+                this,
+              ).filter((element) => element);
+            this.setState({
+              suggestedBindings:
+                candidateBindables as NonDeleted<ExcalidrawBindableElement>[],
+            });
+          });
```

Comment:
this sets the suggested binding state for each arrow element in the selection, meaning it's inefficient and it won't show the bindings for multiple elements at once (if you e.g. drag multiple already-bound arrows).

We can do this instead:

```ts
const suggestedBindings = selectedElements
  .filter(isLinearElement)
  .reduce((acc: NonDeleted<ExcalidrawBindableElement>[], element) => {
    const candidateBindables =
      getOriginalBindingsIfStillCloseOfLinearElement(
        element,
        elementsMap,
        this,
      ).filter(
        (element): element is NonDeleted<ExcalidrawBindableElement> =>
          !!element,
      );
    return acc.concat(candidateBindables);
  }, []);

this.setState({
  suggestedBindings,
});
```